Humpback/Humpback.csproj

```diff
@@ -1,20 +1,36 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">x86</Platform>
-    <ProductVersion>8.0.30703</ProductVersion>
-    <SchemaVersion>2.0</SchemaVersion>
-    <ProjectGuid>{2EA42E6C-D69E-46DD-AA57-5BDC47B1AFF3}</ProjectGuid>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{E7FEAF88-D6C7-4804-9950-E5F21C1D5811}</ProjectGuid>
     <OutputType>Exe</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Humpback</RootNamespace>
-    <AssemblyName>Hump</AssemblyName>
```

Comment:
The expected assemble executable is hump.exe
This may need to be reverted.

Reply:
Removed in commit: 35668189378d8290ca39251b007f30faf8132ed2

Reply:
> The expected assemble executable is hump.exe This may need to be reverted.
Updated assembly name.
![image](https://github.com/PreViser/Humpback/assets/125292739/905d2f7f-6f04-403c-82b6-9c63b1f000b9)